iothub/device/samples/PnpDeviceSamples/Thermostat/Program.cs

```diff
@@ -27,7 +28,21 @@ public class Program
         // DTDL interface used: https://github.com/Azure/opendigitaltwins-dtdl/blob/master/DTDL/v2/samples/Thermostat.json
         private const string ModelId = "dtmi:com:example:Thermostat;1";
 
+        // This environment variable indicates if DPS or IoT Hub connection string will be used to provision the device.
+        // Expected values: (case-insensitive)
```

Comment:
shouldn't we take care of casing?

Reply:
no, it doesn't matter; this is only a switch to indicate which flow is to be demonstrated.

Reply:
you are already doing it actually.